NOMIS/nomis.go

```diff
@@ -320,8 +333,11 @@ func downloadFile() {
 	}
 	defer resp.Body.Close()
 	size, err := io.Copy(file, resp.Body)
+	if err != nil {
+		fmt.Printf("Error copying a file %s", err)
```

Comment:
Capital being used in an error message. Also, we should be using log.Event rather than fmt.Printf